src/components/CategoryCard.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div
+    class="flex flex-column bg-primary shadow-md shadow-primary rounded-md overflow-hidden relative hover:-translate-y-1 hover:shadow-2xl transition-all duration-200 ease-in-out">
+    <router-link :to="'categories/' + getSlug(category.name)">
```

Comment:
Here you can use the modern string interpolation instead of concatenation: `categories/${getSlug(category.name}`